autocomplete.js

```diff
@@ -14,20 +11,31 @@ function createAwsAutocompleteFunction(
       throw new Error(`Failed to list ${outputDataPath.toLowerCase()}: ${error.message || JSON.stringify(error)}`);
     });
 
-    const mappedAutocompleteItems = fetchResult.map((fetchedItem) => {
-      const autocompleteValue = valuePath ? fetchedItem[valuePath] : fetchedItem;
-      const autocompleteLabel = labelPath ? fetchedItem[labelPath] : autocompleteValue;
-      return awsPluginLibrary.autocomplete.toAutocompleteItemFromPrimitive(
-        autocompleteValue,
-        autocompleteLabel,
-      );
-    });
+    const mappedAutocompleteItems = fetchResult.map((fetchedItem) => (
```

Comment:
`Array.map` accepts functions and `awsPluginLibrary.autocomplete.toAutocompleteItemFromPrimitive` is a function already.